docker-app/qfieldcloud/core/views/users_views.py

```diff
@@ -1,17 +1,22 @@
-from django.contrib.auth import get_user_model
 from django.core.exceptions import ObjectDoesNotExist
 from drf_spectacular.utils import extend_schema, extend_schema_view
+
+
 from qfieldcloud.core import pagination, permissions_utils, querysets_utils
-from qfieldcloud.core.models import Organization, Project
+from qfieldcloud.core.models import (
+    Organization,
+    Project,
+)
 from qfieldcloud.core.serializers import (
     CompleteUserSerializer,
     OrganizationSerializer,
     PublicInfoUserSerializer,
 )
+
 from rest_framework import generics, permissions
 from rest_framework.response import Response
 
-User = get_user_model()
+from django.contrib.auth.models import User
```

Comment:
Why you change this? It is not by coinsidence like this. Check the documentation of `get_user_model`  https://docs.djangoproject.com/en/4.2/topics/auth/customizing/#django.contrib.auth.get_user_model

Reply:
Back to the initial state!